src/mdx/expandVersionedUrl.js

```diff
@@ -17,22 +17,7 @@
  */
 
 /** @type {Array<VersionMapping>} */
-const versionMappings = [
-  // 👋 When cutting a new version, add a new mapping here!
-  {
-    docsVersion: "8.2",
-    optimizeVersion: "3.10.0",
-  },
-  {
-    docsVersion: "8.1",
-    optimizeVersion: "3.9.0",
-  },
-  {
-    docsVersion: "8.0",
-    optimizeVersion: "3.8.0",
-  },
-  { docsVersion: "1.3", optimizeVersion: "3.7.0" },
-];
+const versionMappings = [{ docsVersion: "1.3", optimizeVersion: "3.7.0" }];
```

Comment:
The "newest" version gets treated by this MDX plugin as "the one where there are no version numbers in the URLs." By removing all other versions, the plugin gives us the correct URLs when bouncing between main & optimize instances.